### PR TITLE
#654. Add pwd-compose.yml for play with docker. Add PWD button

### PR DESCRIPTION
**Pull request by alfredleo**

### CODE REVIEW

**Comment by alfredleo** [comment]
You need to wait till composer install finishes in background. Watch for CPU to go less then 10%. It takes about 140 seconds to spin up after instance run.

**Comment by alfredleo** [comment]
Hi @XzAeRo. Any problem with merging this? That pr adds PWD support for the project.

**Comment by XzAeRo** [comment]
No problem in the meantime. It's just that I've been very busy last few weeks.

I will review and merge it as soon as I have some spare time.

File: pwd-compose.yml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+      - REPO_LOCAL=/data/git/repos/yii2-starter-kit
+      - REPO_BRANCH=master
+      - REPO_REMOTE=https://github.com/yii2-starter-kit/yii2-starter-kit.git
+    tty: true
+    volumes:
+      - starter_kit_app:/data/git/repos/yii2-starter-kit
+      - mysql_config:/data/git/repos/yii2-starter-kit/docker/mysql
+
+  app:
+    image: alfredleo/starter_kit_app:secondtry
```
Comment (XzAeRo) [inline pwd-compose.yml]: This section is the only thing that troubles me.

Is it using an image that you're building with an external build system?

**Pull request closed without merging**